pimpmycause/pimpmycause/templates/index.html

```diff
@@ -353,7 +353,7 @@ <h4 class="pnl__title">Sign-up to our newsletter</h4>
                                         <input type="email" name="email" placeholder="Email Address" required>
                                     </div>
                                     <div class="medium-6 large-6 small-12 columns">
-                                        <input class="button" type="submit" value="Singup">
+                                        <input class="button" type="submit" value="Sign-up">
```

Comment:
After some furious googling seems it is actually "Sign up" :D Do you mind making the small change?